### PR TITLE
docs(contributingmd): add run tests commands

### LINKED ISSUE #1119: [Docs] Add a How-to-run-tests section into Contributing Guide
### What type of enhancement is this?

Other

### What does the enhancement do?

From our contributor @masonyc, 
> I think it's beneficial to put the how-to-run-tests guides into the github contribute page...

That's a great idea!

### Implementation challenges

_No response_

### PR DESCRIPTION
**Pull request by morigs**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Add description on how to run tests

## Checklist

- [NA]  I have written the necessary rustdoc comments.
- [NA]  I have added the necessary unit tests and integration tests.

## Refer to a related PR or issue link (optional)

Resolves #1119

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ GreptimeDB uses the [Apache 2.0 license](https://github.com/GreptimeTeam/greptim
 
 - To ensure that community is free and confident in its ability to use your contributions, please sign the Contributor License Agreement (CLA) which will be incorporated in the pull request process.
 - Make sure all your codes are formatted and follow the [coding style](https://pingcap.github.io/style-guide/rust/).
-- Make sure all unit tests are passed.
+- Make sure all unit tests are passed (using `cargo test --workspace` or nextest `cargo nextest run`).
```
Comment (killme2008) [inline CONTRIBUTING.md:53]: What about adding a link to nextest? https://nexte.st/index.html

```suggestion
- Make sure all unit tests are passed (using `cargo test --workspace` or [nextest](https://nexte.st/index.html) `cargo nextest run --workspace`).
```

Comment (morigs) [reply CONTRIBUTING.md:53]: Link is a good idea. However, per [nextest docs](https://nexte.st/#quick-start) `--workspace` flag is not needed. Could you pls edit your suggestion?)

Comment (killme2008) [reply CONTRIBUTING.md:53]: Fixed it.

**Review by killme2008 — APPROVED** [review]
LGTM

**Pull request merged**